src/Uno.UI/UI/Xaml/Automation/Peers/RadioButtonAutomationPeer.cs

```diff
@@ -1,57 +1,92 @@
+using System;
 using Uno;
+using Windows.UI.Xaml.Automation.Provider;
+using Windows.UI.Xaml.Controls;
 
 namespace Windows.UI.Xaml.Automation.Peers
 {
-	public  partial class RadioButtonAutomationPeer : ToggleButtonAutomationPeer, Provider.ISelectionItemProvider
+	/// <summary>
+	/// Exposes RadioButton types to Microsoft UI Automation.
+	/// </summary>
+	public partial class RadioButtonAutomationPeer : ToggleButtonAutomationPeer, ISelectionItemProvider
 	{
-		public RadioButtonAutomationPeer(Controls.RadioButton owner) : base(owner)
+		/// <summary>
+		/// Initializes a new instance of the RadioButtonAutomationPeer class.
+		/// </summary>
+		/// <param name="owner"></param>
+		public RadioButtonAutomationPeer(RadioButton owner) : base(owner)
 		{
 		}
 
-		protected override string GetClassNameCore()
+		protected override object GetPatternCore(PatternInterface patternInterface)
```

Comment:
Breaking change ?

Reply:
The `GetClassNameCore` method is below, this one is new, but it should not break any behavior

Reply:
should it be `private protected` then ?

Reply:
The base method has `protected` only, so this should be ok.